### ISSUE TITLE
[game/jail] Prison Change / DoC

### ISSUE THREAD

**Issue by CreamSlush** [InZidiuZ/op-framework-issue-tracker#1748]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes

**Summary:**
Write a summary of what your suggestion is about.
With the new interior that was added into the jail (but of course not usable yet), I'm wanting to see if a couple changes could be implemented to the jail so it could start being opened up. The yard needs to be completely gated off so they are not able to leave the yard area. Same with the staircases around the prison yard. With that being blocked off, the range of how far prisoners could go would need to be increased so they are able to reach the visitation area in the prison. With that being said, I'm wanting to elevate the RP in prison overall with this and start looking at the introduction of the Department of Corrections being introduced where their main job is obviously provide RP in the prison. They would need access to lock/unlock all the doors in the prison. The addition of storage/locker would also be requested to store citizen's belongings if coming to visit an inmate. There are many other changes I think could benefit the RP of the prison, but I do think it's best to start small and request these starting changes since they would be the only things needed to get it started. I can add pictures of the areas that would need to be blocked off in the yard if needed. 

TLDR - 
Requesting the prison yard to be sealed so inmates can't leave it, a storage/locker placed at the citizen area to conduct searches and place items not allowed in the visitation area into, and the final addition to bring it altogether the DoC job which would include (Warden, Captain, Sergeant, Senior Correction Officer, Correction Officer, and Correction Cadet).

**Reason:**
Why would your suggestion be a good change to the framework?
The reasoning to these changes is to make it not feel like going to prison is a soft ban. I feel like a lot of people don't do crime as much as they use to do due to the prison providing no RP for the player and it's almost like you're being put through a soft ban for providing crime RP. With the implementation of these changes would fix a lot of those issues. The prison interior already creates so much more RP and life to the prison if opened up, and with the addition of bringing DoC to the server would just provide even more RP for the criminals to go off of. I don't think it's right that people avoid doing crime sometimes because they feel like it's a soft ban to go to prison. A lot of times I just see them get off the server if they were sent in for a long time also. The prison system currently isn't healthy for the server and has the potential to be so much more.

**Comment by InZidiuZ** [comment]
> With the new interior that was added into the jail (but of course not usable yet)

It is very usable.

> The yard needs to be completely gated off so they are not able to leave the yard area. Same with the staircases around the prison yard.

This is something you would have to redirect to the [assets issue tracker](https://github.com/InZidiuZ/legacy-assets-issue-tracker/issues).

> They would need access to lock/unlock all the doors in the prison.

This is already available. You just need the "Law Enforcement" job.

> The addition of storage/locker would also be requested to store citizen's belongings if coming to visit an inmate.

This is also already available when you go on-duty behind the desk of the reception.

> DoC job which would include (Warden, Captain, Sergeant, Senior Correction Officer, Correction Officer, and Correction Cadet).

A job dedicated for the prison is not something that has been added yet as the current systems are a mess and is meant to be rewritten in the future to allow for more complex and clean features surrounding the prison. Right now though, some temporary systems have been set up that should make prison RP possible, though not super optimal.

**Comment by CreamSlush** [comment]
> > With the new interior that was added into the jail (but of course not usable yet)
> 
> It is very usable.

There are a couple of issues I've seen with it so far, as in falling through some places and what not, would that also need to be redirected with the yards to be gated off?

**Comment by InZidiuZ** [comment]
I think that's collision issues that can be fixed. Though they should be reported in the [assets issue tracker](https://github.com/InZidiuZ/legacy-assets-issue-tracker/issues).

**Comment by CreamSlush** [comment]
So, I've been talking to Nuzi about some stuff. Besides jail breaks and what not, he wanted me to take a video of the locations where we will need to a check in option for the Prison and where I would want an evidence locker to store visitor's items. (I know it was discussed that there is a locker already, but those are private lockers. It would be good to have an actual evidence locker at the front to put visitor's items that anyone could retrieve back for them when they decide to leave.)
https://www.youtube.com/watch?v=X-U0w009jag

**Comment by Nuziatella** [comment]
Department of Corrections is kicking off but they need to wait until a few things get sorted by you:
Jobs:

```
		["Bolingbroke Penitentiary"] = {
			["Cadet"] = {salary = 55}
                        ["Guard"] = {salary = 70}
                        ["Senior Guard"] = {salary = 82}
                        ["Head Guard"] = {salary = 96}
                        ["Vice Warden"] = {salary = 134}
                        ["Warden"] = {salary = 152}
		},

```
Radio: Channels 2 and 3
They need to be able to use these channels without going on-duty to not add to the total cop count for the sake of robberies. Unless you can have them go on duty but exclude the department of Bolingbroke Penitentiary from the robbery count. 

Store infront of kitchen with hamburger and water in it

**Comment by CreamSlush** [comment]
With DOC coming to an approach on being completely open, One of my bigger concerns that I'm hoping could be resolved is getting a check in option for the prison in the medbay. COs can cpr someone up, but it does not fix injuries. EMS and Doctors can be called to the prison, but prisoners can't leave the prison to go physically check in to remove their head injury. They're forced to be with that injury. Since I'm staff, if I'm there I can revive them to remove these injuries, but I won't be there at all times. With possibly more people getting higher jail times and this opening more options to actually have someone held till trial more often, this would be a huge quality of life mechanic to just have the check in option there.

**Comment by InZidiuZ** [comment]
Closing as outdated. If this is still relevant, please create a new issue.
